docs/tutorials/python/table.md

```diff
@@ -204,6 +204,11 @@ Qux2,4,203001,204001,+,False
     table.delete_rows(query=f"SELECT * FROM {table.id} WHERE Strand = '+'")
     ```
 
+* Or deleting rows based on a filtered dataframe, where the ROW_ID and ROW_VERSION columns specify the rows to be deleted from the table. In this example, rows 2 and 3 are deleted.
```

Comment:
Is row version the version of the entity/Synapse object, or can rows have their own versions on Synapse? What would be the value in changing, version 1 of a row, for example, if it's latest version is 3? I think I need more clarity on what this `row_version` arg is for, thanks in advance!